HOMEWORKS/Chapter_02/Rasha/RashaHomework2/assets/script/javascript.js

```diff
@@ -0,0 +1,15 @@
+let Radius = 5 ;
+let pi =3.14 ;
+ let surfaceArea = 4 * pi * ( Radius *Radius);
+
+ if (surfaceArea >= 313.5 && surfaceArea <= 314) {
+  
+
+    let htmlMessage = document .getElementById("Calculate");
```

Comment:
```suggestion
    let htmlMessage = document .getElementById("Calculate");
```

If you declared using `var` keyword then your solution would be good. BUT with let it would be wrong.
Have you test the page ?